src/services/json-api-datastore.service.ts

```diff
@@ -69,6 +69,21 @@ export class JsonApiDatastore {
       .catch((res: any) => this.handleError(res));
   }
 
+  findOne<T extends JsonApiModel>(
```

Comment:
This method is just the same as findRecord, except of one line. Could you make a private "internal" method instead which combines `findOne` and `findRecord` ?

Reply:
Do you mean something like in e971906?